cawemo-to-web-modeler-migration/migrate-cawemo-to-web-modeler.sh

```diff
@@ -0,0 +1,210 @@
+#!/bin/sh 
+
+# ℹ️ Requires Bash 4 or newer, or zsh
+# Recommended to run it on a test environment first
+
+# See these docs to obtain your Cawemo API credentials: https://docs.camunda.org/cawemo/1.9/reference/rest-api/overview/authentication/
+CAWEMO_USER_ID=ENTER_HERE
+CAWEMO_API_KEY=ENTER_HERE
+
+# See these docs to obtain your Web Modeler client credentials: https://docs.camunda.io/docs/next/apis-tools/web-modeler-api/#authentication
+MODELER_CLIENT_ID=ENTER_HERE
+MODELER_CLIENT_SECRET=ENTER_HERE
+
+# Color definitions for terminal output formatting
+GREEN='\033[0;32m'
+NC='\033[0m' # No Color
+B=$(tput bold)
+N=$(tput sgr0)
+BGREEN=$GREEN$B
+NNC=$NC$N
+
+echo "------------------------------------------------------------------------"
+echo "${BGREEN}MIGRATION FROM CAWEMO TO WEB MODELER${NNC}"
+echo "Files are now migrated file-by-file..."
+echo "------------------------------------------------------------------------"
+
+echo ""
+
+echo "[$(date +%s)] Migration started" >> migration.log
+
+HTTP_STATUS=$(curl -s -o .curl_tmp -w "%{http_code}" -u $CAWEMO_USER_ID:$CAWEMO_API_KEY "https://cawemo.com/api/v1/projects")
+
+PROJECTS=$(cat .curl_tmp)
+
+if [ "$HTTP_STATUS" != "200" ]
+then
+    echo "⚠ GET https://cawemo.com/api/v1/projects failed with status code $HTTP_STATUS"
+    echo "[$(date +%s)] ⚠ GET https://cawemo.com/api/v1/projects failed with status code $HTTP_STATUS" >> migration.log
```

Comment:
You could maybe use a more user-friendly date format (e.g. `date +'%Y-%m-%d %H:%M:%S'`) for all the logging statements.